bot/cogs/tickets.py

```diff
@@ -0,0 +1,405 @@
+from __future__ import annotations
+
+from functools import lru_cache
+from typing import TYPE_CHECKING, NamedTuple, Optional, Union
+
+import asyncpg
+import discord
+from discord.ext import commands
+from discord.utils import format_dt, utcnow
+from libs.tickets.structs import ReservedTags, StatusChecklist, TicketThread
+from libs.tickets.utils import get_cached_thread, get_partial_ticket
+from libs.utils.embeds import Embed, LoggingEmbed
+
+from .config import GuildWebhookDispatcher
+
+if TYPE_CHECKING:
+    from libs.utils import RoboContext
+    from rodhaj import Rodhaj
+
+
+STAFF_ROLE = 1184257456419913798
```

Comment:
Shouldn't this be configurable?

Reply:
The short answer is no. It would be more hassle in order just to pre-populate and track the staff role when in reality, the staff roles will not change any time soon